client/state/data-layer/wpcom-api-middleware.js

```diff
@@ -98,13 +98,14 @@ export const middleware = handlers => store => next => {
 		// this guarantees that we don't double-dispatch
 		const nextActions = new Set();
 		const safeNext = a => nextActions.add( a );
-
-		handlerChain.forEach( handler => handler( store, action, safeNext ) );
+		let shouldIgnore = false;
```

Comment:
Sorry :$ at the time it looked like the way with less code changes to get a discard mechanism, and I was thinking of better ways to do it. I will avoid hurting your middleware again :)